chart/values.yaml

```diff
@@ -129,7 +129,7 @@ shell:
 
   # Persistent storage for SSH fingerprint keys
   storage: 1M
-  storageClassName: silta-shared
+  storageClassName: standard
```

Comment:
Rather than hard-coding "standard" (which is the default for Google Cloud but might not be elsewhere), we should leave that undefined, meaning using the default. See https://github.com/helm/charts/blob/d05341dea09eca4a88177988355b1a2af7abace9/stable/minio/values.yaml#L74 for a good explanation.

Reply:
Variables and definition removed